manim/utils/cfg_subcmds.py

```diff
@@ -154,8 +154,14 @@ def write(level=None, openfile=False):
                         else default[key]
                     )
                     console.print(f"(defaults to {defaultval}) :", end="")
-
-                temp = input()
+                try:
+                    temp = input()
+                except EOFError:
+                    raise Exception(
+                        """Not enough values in input.
+If you have added a new entry to default.cfg, have you modified _write_cfg_subcmd_input
+to account for it?"""
```

Comment:
I'd prefer
"You may have added a new entry to default.cfg, in which case you will have to modify _write_cfg_subcmd_input to account for it."

Reply:
Done :)